main.js

```diff
@@ -59,11 +63,13 @@ function addPassword(service, password) {
   passwordValue.size = 50;
   passwordValue.setAttribute('autocomplete', 'off');
   passwordCell.appendChild(passwordValue);
+  passwordValue.className = 'form-control';
```

Comment:
This is overriding the monospace attribute which is also helpful for manual entry, can we combine the two here?

Reply:
Yeah, we could write a single class and use the CSS from both or have one that has all of its desired attributes.

Reply:
I think what would be ideal is to remove line 60 and replace this line with `passwordValue.className = 'form-control monospace;`